Formatter/Pool.php

```diff
@@ -12,14 +12,16 @@
 namespace Sonata\FormatterBundle\Formatter;
 
 use Psr\Log\LoggerAwareInterface;
-use Psr\Log\LoggerInterface;
+use Psr\Log\LoggerAwareTrait;
 use Psr\Log\NullLogger;
 use Twig_Environment;
 use Twig_Error_Syntax;
 use Twig_Sandbox_SecurityError;
 
 class Pool implements LoggerAwareInterface
```

Comment:
LoggerAware can be used as a marker that it requires a log, so I would say keep it. Doesn't hurt.